package.json

```diff
@@ -30,28 +29,26 @@
     "test:update": "npm test -- --updateSnapshot --coverage",
     "test:debug": "node --inspect-brk ./node_modules/.bin/jest --watch --runInBand",
     "validate": "kcd-scripts validate",
-    "setup": "npm install && npm run validate -s",
-    "dtslint": "dtslint typings"
+    "setup": "npm install && npm run validate -s"
   },
   "husky": {
     "hooks": {
       "pre-commit": "kcd-scripts pre-commit"
     }
   },
   "files": [
-    "dist",
-    "typings"
+    "dist"
   ],
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@sheerun/mutationobserver-shim": "^0.3.2",
+    "@types/testing-library__dom": "^6.0.0",
```

Comment:
Ok, so I've added this as a dep. Here's what things look like before any of these changes (with the current 6.0.0):

<img width="868" alt="Screen Shot 2019-08-15 at 1 01 14 PM" src="https://user-images.githubusercontent.com/1500684/63119241-faf88c00-bf5c-11e9-83a9-c39e91b582fb.png">

And here's what it looks like before I added this dependency (with 6.0.1-alpha.0):

<img width="670" alt="Screen Shot 2019-08-15 at 1 00 12 PM" src="https://user-images.githubusercontent.com/1500684/63119268-08157b00-bf5d-11e9-9570-c6204614b0dd.png">

And here's what it looks like after I added this dependency (with 6.0.1-alpha.1):

<img width="737" alt="Screen Shot 2019-08-15 at 1 00 32 PM" src="https://user-images.githubusercontent.com/1500684/63119287-12377980-bf5d-11e9-8f4f-3ea3aacc5b81.png">

So as far as the consumer is concerned, there's absolutely no change. And if someone wants to have more control over the version of the typings, then they can install `@types/testing-library__dom` themselves and `npm` or `yarn` will de-dupe to whatever version they want.

This is getting a 👍 from me. Let's do it!